src/anm/utils.js

```diff
@@ -12,7 +12,7 @@ is.defined = function(v) {
 is.finite = global.isFinite;
 is.nan = global.isNaN;
 is.arr = Array.isArray;
-is.int = function(n) {
+is.integer = function(n) {
```

Comment:
All functions name inside `is` object consist of three-letter (okay, except `is.defined`, `is.finite` and `is.modifier` :) ), I think then it's better to rename them all to full names, for the sake of consistency.